<launch>
    <arg name="frame_rate" default="30"/>
    <arg name="img_width" default="1280" />
    <arg name="img_height" default="960" />

    <node name="camera" pkg="usb_cam" type="usb_cam_node" output="screen">
        <param name="video_device" value="/dev/video0" />
        <param name="image_width" value="$(arg img_width)" />
        <param name="image_height" value="$(arg img_height)" />
        <param name="framerate" value="$(arg frame_rate)" />
        <param name="pixel_format" value="yuyv" />
        <param name="camera_name" value="overhead_cam" />
    </node>
</launch>
